packages/dotcom-ui-header/src/__test__/components/__snapshots__/StickyHeader.spec.tsx.snap

```diff
@@ -834,7 +832,27 @@ exports[`dotcom-ui-header/src/components/StickyHeader renders as an anonymous us
         </div>
         <div
           className="o-header__top-column o-header__top-column--right"
-        />
+        >
+          <a
+            className="o-header__top-button o-header__top-button--hide-m"
+            data-trackable="Subscribe"
+            href="/products?segmentId=#"
+            role="button"
+            tabIndex={-1}
+          >
+            Subscribe
+          </a>
+          <a
+            className="o-header__top-myaccount"
+            data-trackable="Sign In"
+            href="/login?location=#"
+            id="o-header-top-link-signin"
+          >
+            <span>
+              Sign In
+            </span>
+          </a>
+        </div>
```

Comment:
**Review notes:** This appears to have been missing from the original snapshot output. Based on my investigation I am not sure why 🤷‍♂️ Looking at the code and also the experience of the sticky header on production I believe its correct that these should be shown in this sceneario (Anon user)

Reply:
Ok, I now have an answer for this (I missed something in my original investigation that a nights sleep helped with)

The fixture data that is passed into the test sets `userIsLoggedIn` to `false` and `showUserNavigation` to `false` which means that [this if statement](https://github.com/Financial-Times/dotcom-page-kit/blob/main/packages/dotcom-ui-header/src/components/sticky/partials.tsx#L153-L157) never matches and therefore renders nothing within the top right nav.

On ft.com I don't believe that this is a scenario we actually use, when a user is anonymous we still show right column nav to prompt the user to sign in or subscribe. So I feel like my changes are now a better representation of what we use in production